Makefile

```diff
@@ -249,6 +250,11 @@ can-build-pr:
 		exit 1; \
 	fi
 
+i18n-extract: ## Extract strings for translation from the source code
+	@[[ -d $(MM_UTILITIES_DIR) ]] || echo "You must clone github.com/mattermost/mattermost-utilities repo in .. to use this command"
+	@[[ -d $(MM_UTILITIES_DIR) ]] && cd $(MM_UTILITIES_DIR) && npm install && npm run babel && node mmjstool/build/index.js i18n extract-mobile
```

Comment:
can I assume that extract-mobile will update the en.json file under `assets/base/i18n` ?

Reply:
yes, exactly, concretely the file `assets/base/i18n/en.json`